test/unit/services/JobRoleServiceTests.ts

```diff
@@ -19,6 +20,19 @@ const jobRoleResponse: JobRoleResponse = {
     formattedLocations: "Birmingham"
 }
 
+const getJobRoleByIdResponse: JobRole = {
+  roleId: 5,
+  roleName: "Technical Architect",
+  description: "Test description for the role",
+  linkToJobSpec: "examplelink.co.uk",
+  responsibilities: "One, two, three, ninety-nine",
+  locations: Location.Birmingham,
```

Comment:
Remove the pluralisation.

Reply:
locations is actually a string that returns multiple locations (Comment has been overridden by Tom)